### PR TITLE
chore: Added Quine Resources

### LINKED ISSUE #1188: [Add] Quine Resources
### Site URL

https://www.youtube.com/playlist?list=PLnCV5fTAsUgpaLMY_Z8FqPxZ89v-_pcfY

### Description of the above provided site

- Quine is a open source platform where user can add project and rank your project by getting likes and comments.
- Quine provide a platform where any newbie contributor can go for the `good first issue` and enter into the open source world.
- This playlist will help to those contributor who want to explore quine or new on the quine platform.
- Having a best content with best guidance.

### Category

Open Source

### Sub Category

Videos

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Avdhesh-Varshney**
## Fixes Issue - 1188

Closes #1188 

## Changes proposed

Added quine resources

https://www.youtube.com/playlist?list=PLnCV5fTAsUgpaLMY_Z8FqPxZ89v-_pcfY

### CODE REVIEW

**Comment by Avdhesh-Varshney** [comment]
@rupali-codes @CBID2 please review this pr

**Review by CBID2 — APPROVED** [review]
LGTM!

File: database/open_source/blogs.json [hunk 1]
```diff
@@ -26,5 +26,12 @@
     "url": "https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github",
     "category": "open source",
     "subcategory": "open-source-blogs"
+  },
+  {
+    "name": "How to contribute in Quine Open source platform",
+    "description": "Are you a beginner in open source? This site will help you in making your first open-source contribution, teach you about the Quine open source platform, and connect with other open source contributors.",
+    "url": "https://www.youtube.com/playlist?list=PLnCV5fTAsUgpaLMY_Z8FqPxZ89v-_pcfY",
+    "category": "opensource",
+    "subcategory": "opensource_blogs"
```
Comment (rupali-codes) [inline database/open_source/blogs.json:35]:
```suggestion
    "subcategory": "open-source-blogs"
```

File: database/open_source/blogs.json [hunk 2]
```diff
@@ -26,5 +26,12 @@
     "url": "https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github",
     "category": "open source",
     "subcategory": "open-source-blogs"
+  },
+  {
+    "name": "How to contribute in Quine Open source platform",
+    "description": "Are you a beginner in open source? This site will help you in making your first open-source contribution, teach you about the Quine open source platform, and connect with other open source contributors.",
+    "url": "https://www.youtube.com/playlist?list=PLnCV5fTAsUgpaLMY_Z8FqPxZ89v-_pcfY",
+    "category": "opensource",
```
Comment (rupali-codes) [inline database/open_source/blogs.json:34]:
```suggestion
    "category": "open source",
```

**Review by k-deepak04 — CHANGES_REQUESTED** [review]
do the needful naming convention cahnges suggested

**Review by k-deepak04 — APPROVED** [review]

**Comment by k-deepak04** [comment]
@rupali-codes plz review and merge this

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Pull request merged**